src/Compilers/CSharp/Test/Syntax/Parsing/ParsingErrorRecoveryTests.cs

```diff
@@ -8110,5 +8110,109 @@ public void RazorCommentRecovery_NoStart()
             }
             EOF();
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/75583")]
+        public void PreprocessorDirective_Trailing_01()
+        {
+            UsingTree("""
+                if (#if)
```

Comment:
can you have variants with things like whitespace after the `(` and before/after the `#`.  i want to make sure all the positions/text are correct for all the tokens.